src/alfasim_score/converter/alfacase/score_output_generator.py

```diff
@@ -31,30 +31,35 @@ def __init__(
         results_path: Path,
         well_start_position: Scalar,
         active_annuli: List[AnnulusLabel] = [],
+        walls: List[int] = [],
     ):
         self.results_path = results_path
         self.well_start_position = well_start_position
         self.active_annuli = active_annuli
+        self.walls = walls
         self.element_name = WELLBORE_NAME
 
     def _generate_output_results(self) -> Dict[str, Any]:
         """Create data for the output results."""
+        results = Results(self.results_path)
+        measured_depths = self.well_start_position.GetValue(LENGTH_UNIT) + np.array(
+            results.get_profile_curve("pressure", self.element_name, -1).domain.GetValues("m")
```

Comment:
Is there a `MEASURE_UNIT` to replace `"m"`?

Reply:
Yeah, changed for `LENGTH_UNIT`.